### PR TITLE
Eliminate pipes chapter

### PR DESCRIPTION
**Pull request by hadley**
I've included the most important content in the transform chapter. And suggested we do the comparison in a blog post: https://github.com/tidyverse/tidyverse.org/issues/626

### CODE REVIEW

File: data-transform.qmd [hunk 1]
```diff
@@ -71,8 +71,8 @@ But before we discuss their individual differences, it's worth stating what they
 
 3.  The result is always a new data frame.
 
-Because the first argument is a data frame and the output is a data frame, dplyr verbs work well with the pipe, `|>`.
-The pipe takes the thing on its left and passes it along to the function on its right so that `x |> f(y)` is equivalent to `f(x, y)`, and `x |> f(y) |> g(z)` is equivalent to into `g(f(x, y), z)`.
+Since each verb is quite simple, solving complex problems will usually require combining multiple verbs, and we'll do so with the pipe, `|>`.
+We'll come back to discuss the pipe more in @the-pipe, but in brief, the pipe takes the thing on its left and passes it along to the function on its right so that `x |> f(y)` is equivalent to `f(x, y)`, and `x |> f(y) |> g(z)` is equivalent to into `g(f(x, y), z)`.
```
Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:75]: Just above we said "we'll come back", to say again "we'll come back" sounds a bit odd to me.

File: data-transform.qmd [hunk 2]
```diff
@@ -87,11 +87,8 @@ flights |>
   )
 ```
 
-The code starts with the `flights` dataset, then filters it, then groups it, then summarizes it.
-We'll come back to the pipe and its alternatives in @sec-pipes.
-
 dplyr's verbs are organized into four groups based on what they operate on: **rows**, **columns**, **groups**, or **tables**.
-In the following sections you'll learn the most important verbs for rows, columns, and groups, then we'll come back to verbs that work on tables in @sec-joins.
+In the following sections you'll learn the most important verbs for rows, columns, and groups, then we'll come back to the job verbs that work on tables in @sec-joins.
```
Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:91]: Should this say "the join" verbs?

File: data-transform.qmd [hunk 3]
```diff
@@ -290,9 +279,11 @@ flights |>
 ```
 
 Alternatively, you can control which variables are kept with the `.keep` argument.
-A particularly useful argument is `"used"` which allows you to see the inputs and outputs from your calculations:
+A particularly useful argument is `"used"` which allows you to see the inputs and outputs from your calculations.
+For example, the followingc output will contain only the variables `dep_delay`, `arr_delay`, `air_time`, `gain`, `hours`, and `gain_per_hour`.
```
Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:283]:
```suggestion
For example, the following output will contain only the variables `dep_delay`, `arr_delay`, `air_time`, `gain`, `hours`, and `gain_per_hour`.
```

File: data-transform.qmd [hunk 4]
```diff
@@ -372,6 +377,8 @@ flights |>
 But you can use the same `.before` and `.after` arguments as `mutate()` to choose where to put them:
 
 ```{r}
+#| results: false
```
Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:380]: I know we're hiding the code output to save space, but it's sad that code and output can't always be presented, it's much nicer when we can do that.

**Review by mine-cetinkaya-rundel — APPROVED** [review]
Just a couple of minor things.

Also, should workflow-pipes.qmd be deleted then? (There's one paragraph still sitting there.)

File: data-transform.qmd (hunk 2 above)
Comment (hadley) [reply data-transform.qmd:91]: 😭

File: data-transform.qmd (hunk 4 above)
Comment (hadley) [reply data-transform.qmd:380]: Agreed

**Pull request merged**